ldes-server-port-fetch-rest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/rest/treenode/TreeNodeController.java

```diff
@@ -57,10 +60,14 @@ public ResponseEntity<ResponseBodyEmitter> retrieveLdesFragmentStreaming(@PathVa
 																			 @RequestParam Map<String, String> requestParameters,
 																			 @PathVariable("collectionname") String collectionName) {
 		final ViewName viewName = new ViewName(collectionName, view);
+		LdesFragmentIdentifier id = new LdesFragmentIdentifier(viewName, requestParameters.entrySet()
+				.stream()
+				.map(entry -> new FragmentPair(entry.getKey(), entry.getValue()))
+				.toList());
```

Comment:
I'd propose either making the tabbing more clear or declaring/assigning the second parameter in a separate line for readability